recipes/libMobility/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "libmobility" %}
+{% set version = "0.5.0" %}
+
+{% set cuda_compiler_version = cuda_compiler_version | default("None") %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/stochasticHydroTools/libMobility/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 44a3298709b8d8ae9ef16b0c49167510a3d49dafe5f4d4dfd6670f365004959c
+
+build:
+  skip: true  # [win or not (cuda_compiler_version or "").startswith("12")]
+  number: 0
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}blas{{ blas_impl }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+  rpaths:
+    - lib/
+  missing_dso_whitelist:
+    - '*/libcuda.*'  # [linux64]
+  script_env:
+    - CMAKE_ARGS+="-DMKL_INCLUDE_DIR=${PREFIX}/include"  # [blas_impl == "mkl"]
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - {{ stdlib("c") }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - make
+    - cmake >=3.23
+    - numpy
+    - nomkl                 # [blas_impl != "mkl"]
+  host:
+    - python
+    - numpy
+    - cuda-version {{ cuda_compiler_version }}
+    - cuda-cudart-dev
+    - libcufft-dev
+    - libcublas-dev
+    - libcusolver-dev
+    - libcurand-dev
```

Comment:
The link checker reports that these libraries are unused at runtime, but the logs show they are linked at build time. Perhaps you are overlinking (update build flags in a future release) or perhaps you are only using headers (add the runtime libraries to the ignore_run_exports section). I know that curand has header-only components, but I'm not sure that cusolver does.